pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -198,9 +194,6 @@ def on_evaluation_batch_end(
 
         self.trainer.logger_connector.on_batch_end()
 
-        # store predicitons if do_write_predictions and track eval loss history
-        self.store_predictions(output, batch_idx, dataloader_idx)
-
     def store_predictions(self, output: Optional[STEP_OUTPUT], batch_idx: int, dataloader_idx: int) -> None:
```

Comment:
this is dead code now, can also be removed. but there is a call there to the dev debugger which we need to pull up and keep.

Reply:
couldn't push to your branch due to permission error, but removing lines 197 - 204 will fix the tests

Reply:
thanks @awaelchli !